### PR TITLE
Add Binary Search in python3 #92

### PR DESCRIPTION
**Pull request by KaveeshBaddage**

### CODE REVIEW

File: binary_search/binarySearch.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#Binary Search in python3
+#Define an array with elements
+array=[2,3,5,7,11,13,17,19,23,27,29]
+startingIndex=0
+lastIndex=len(array)-1
+middleIndex=(startingIndex+lastIndex)//2
+print ("Defined array's elements are - "+str(array))
+target=int(input("Enter a value of a element in defined array :"))
+
+
```
Comment (t2013anurag) [inline binary_search/binarySearch.py]: remove all extra lines.

File: binary_search/binarySearch.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+flag=0
+while flag==0:
```
Comment (t2013anurag) [inline binary_search/binarySearch.py]: flag == 0

File: binary_search/binarySearch.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#Binary Search in python3
+#Define an array with elements
```
Comment (t2013anurag) [inline binary_search/binarySearch.py]: add proper spacing between operators and operands.

Comment (KaveeshBaddage) [reply binary_search/binarySearch.py]: Okay.. I will do

File: binary_search/binarySearch.py (hunk 1 above)
Comment (KaveeshBaddage) [reply binary_search/binarySearch.py]: Okay.. I will do

**Comment by KaveeshBaddage** [comment]
Removed unnecessary extra lines and add proper spacing between operators and operands.

File: binary_search/binarySearch.py (hunk 3 above)
Comment (KaveeshBaddage) [reply binary_search/binarySearch.py]: Do I need to remove those first two lines ? @t2013anurag

Comment (t2013anurag) [reply binary_search/binarySearch.py]: Ahh, no. Keep them.

**Pull request merged**